content/applications/sales/point_of_sale/restaurant/tips.rst

```diff
@@ -2,30 +2,35 @@
 Integrate a tip option into payment
 ===================================
 
-As it is customary to tip in many countries all over the world, it is
-important to have the option in your PoS interface.
-
-Configure Tipping
-=================
-
-To activate the *Tips* feature, go to :menuselection:`Point of Sale
---> Configuration --> Point of sale` and select your PoS.
-
-Under the Bills & Receipts category, you will find *Tips*. Select it
-and create a *Tip Product* such as *Tips* in this case.
-
-.. image:: tips/tips01.png
-    :align: center
-
-Add Tips to the bill
-====================
-
-Once on the payment interface, you now have a new *Tip* button
-
-.. image:: tips/tips02.png
-    :align: center
-
-Add the tip your customer wants to leave and process to the payment.
-
-.. image:: tips/tips03.png
-    :align: center
+Tipping is customary in multiple countries. Point of Sale allows tipping in **shops**, **bars** or
+**restaurants**.
+
+Configuration
+=============
+
+To activate the **Tips** feature, go to :menuselection:`Point of Sale --> Configuration --> Point
+of Sale`, select the POS, scroll down to the :guilabel:`Payment` section and enable
+:guilabel:`Tips`. Once enabled, add a :guilabel:`Tip Product` in the corresponding field, and save.
+The referred product will be used as a reference on customer receipts.
+
+.. image:: tips/tips-setup.png
+   :align: center
+   :alt: enable tips in a POS
+
+.. note::
+   - You can create a product on the spot. To do so, enter your product's name and click
+     :guilabel:`Create`. The product is automatically configured, but you can click
+     :guilabel:`Create & edit...` to open the product creation form and set it up manually.
+   - If you are using a payment terminal and wish to enable **tips** using the terminal, check
+     :guilabel:`Add tip through payment terminal`.
+
+Add tips
+========
+
+Once an order is done, click :guilabel:`Payment` to access the payment screen. Click
```

Comment:
```suggestion
Upon payment of the order/When registering the payment in your POS, click :guilabel:`Payment` to access the payment screen. Click
```
Once an order is done (to me) makes it sound as if it is *after* the payment is cashed-in